src/rest_framework_dso/serializers.py

```diff
@@ -170,31 +172,33 @@ def many_init(cls, *args, **kwargs):
 
         return list_serializer_class(*args, **list_kwargs)
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    @property
+    def fields(self):
```

Comment:
Als je toch voor een property gaat, is `get_fields()` overriden beter. Die logica gebruikt DRF ook intern:

DRF sourcecode:
```
    @cached_property
    def fields(self):
        """
        A dictionary of {field_name: field_instance}.
        """
        # `fields` is evaluated lazily. We do this to ensure that we don't
        # have issues importing modules that use ModelSerializers as fields,
        # even if Django's app-loading stage has not yet run.
        fields = BindingDict(self)
        for key, value in self.get_fields().items():
            fields[key] = value
        return fields
```

...en ik zie dat `ModelSerializer.get_fields()` ook de velden aanpast. Dat kunnen wij ook goed toevoegen zo, en is idd beter dan `self.fields` assignen.

Reply:
Ja, dacht ik ook, maar ging voor `fields` property omdat `chached_property` moet gereset worden elke keer we limitieren de velden lijst, en dit gebeurt minimaal 2x per serializer.
 - Bij creatie van de serializer
 - Bij data weergave

En bij eerste staap hebben we nog geen request, die wordt later gezet.